LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2024
+Copyright (c) 2024 Mahdi, Mayito & Uzair
```

Comment:
# Backend review

- [ ] In appointments_controller, what happens if the appointment is not successfully destroyed?  Kindly, add some error handling like your did in the create method (line 11) 👍 

![image](https://github.com/mahdinoori2000/healthcare-app-backend/assets/57421823/20deecf6-5403-4714-b4fb-1de8638a41ab)